src/main/java/frc/robot/subsystems/intake/IntakeLoggedInputs.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.subsystems.intake;
+
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+public class IntakeLoggedInputs implements LoggableInputs {
+    public double motorPower;
+    public double angleMotorAngle;
+
+    /**
+     * Implement the variables inside the table.
+     */
+    @Override
+    public void toLog(LogTable table) {
+        table.put("motor power", motorPower);
+        table.put("angle motor power", angleMotorAngle);
+    }
```

Comment:
Just without spaces in the name

Reply:
Why? I think with spaces it's better, you write your logs in English, not in Java.

Reply:
Because every other subsystem uses camelCase. So I think it would be better to align with the other subsystems' conventions.

Reply:
> Because every other subsystem uses camelCase. So I think it would be better to align with the other subsystems' conventions.

IMO status quo ≠ convention. Conventions need to be set, and it didn't happen.